### PR TITLE
improvements

### PR DESCRIPTION
**Pull request by francescopeloi**
improvements

### CODE REVIEW

File: javascript/step_1.html [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-    <p>For Javascript, our starter project is <a href="https://github.com/pokerwars/pokerwars.io-starterbot-javascript"
-            target="_blank">here</a>. We will build that same starter bot now step by step. At the end, you will get
-        an understanding of the game and the platform, and a working bot ready for you to implement your own unique
-        strategy. How cool is that!
-    </p>
+  <p>Welcome to pokerwars! In this tutorial you will learn how to build a simple bot. Let's get
+    started!</p>
+  <h2>The starter bots</h2>
+  <p>In our <a href="https://github.com/pokerwars" target="_blank">GitHub page</a> you can find
+    starter bots for a number of languages. These are very simple versions of pre-built bots. You
```
Comment (giulioleso) [inline javascript/step_1.html]: what about changing `These are very simple versions of pre-built bots` into `These are very simple pre-build bots`

File: javascript/step_2.js [hunk 2]
```diff
@@ -12,5 +12,4 @@ app.use(bodyParser.urlencoded({
 app.use(bodyParser.json());
 
 // Listen
-
```
Comment (giulioleso) [inline javascript/step_2.js:15]: I don't see any difference between step_2 and the solution one, apart from the comment and a missing new line... I'm confused

File: javascript/step_3.html [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-        track of all the tournaments and calls each bot for the next action. For now, let's not worry about that and
+    <p>Next we need to declare some routes. These are endpoints that our bot will be exposing when it's up and
+        running, and that will be called by the pokerwars Engine! This Engine is where all the magic happens. It keeps
+        track of all the poker tournaments and calls each bot for the next action. For now, let's not worry about that and
         let's focus on how to create routes.
     </p>
     <p>In Express, we can create a new route using
         <i>app.route()</i>. In the code, we went ahead and created a route for the <i>play</i> action for you. This
-        is the endpoint the Engine will call (with a POST request) during the tournament to ask you for your next action
+        is the endpoint the Engine will call (with a POST request) during the tournament to ask you for your next poker action (fold, call, bet or raise)
```
Comment (giulioleso) [inline javascript/step_3.html:10]: this line is quite long, you can break it (sorry just cosmetics)

**Comment by giulioleso** [comment]
in step4 can we change `usernameOfSmallBlindBot` with `smallBlindBotUsername`?

File: javascript/step_2.js (hunk 2 above)
Comment (francescopeloi) [reply javascript/step_2.js:15]: that's correct, there's nothing to do in step 2, that's why they are the same, so they always pass. It's a step so they can train on how to use the UI.

File: javascript/step_3.html (hunk 3 above)
Comment (francescopeloi) [reply javascript/step_3.html:10]: reason I left it long is that intellij fucks everything up if you try to break it down, I would need to reformat the whole file unfortunately

**Review by plopcas — APPROVED** [review]

**Pull request merged**